pubsub/google/cloud/pubsub/iam.py

```diff
@@ -107,125 +101,44 @@ class Policy(object):
     :type version: int
     :param version: unique version of the policy
     """
+    _OWNER_ROLES = (OWNER_ROLE, PUBSUB_ADMIN_ROLE)
+    _EDITOR_ROLES = (EDITOR_ROLE, PUBSUB_EDITOR_ROLE)
+    _VIEWER_ROLES = (VIEWER_ROLE, PUBSUB_VIEWER_ROLE)
+
     def __init__(self, etag=None, version=None):
-        self.etag = etag
-        self.version = version
-        self.owners = set()
-        self.editors = set()
-        self.viewers = set()
+        super(Policy, self).__init__(etag, version)
         self.publishers = set()
         self.subscribers = set()
 
-    @staticmethod
-    def user(email):
-        """Factory method for a user member.
+    def _bind_custom_role(self, role, members):
```

Comment:
This is going to break when custom roles role out, and is already broken for the alpha customers of custom roles. We shouldn't try to keep an accounting of role strings, they can be user defined and change at runtime. If we want to provide shortcuts for "legacy" roles like `roles/owners` then we should implement those as I suggest above